recipes/tl2cgen/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "tl2cgen" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dmlc/treelite/archive/{{ version }}.tar.gz
+  sha256: e5b601e7e43193a50adc17e547002147ed94e30e2e95cf3347379ddc5eb81243
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cmake >=3.14
+    - ninja
+    - fmt
+    - rapidjson
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip >=22.1
+    - treelite>=3.4,<4
```

Comment:
I will remove the constraint `<4` once Treelite 4.0 is released. I plan to make many breaking changes to Treelite 4.0.